packages/components/src/focal-point-picker/index.js

```diff
@@ -182,7 +180,7 @@ export class FocalPointPicker extends Component {
 		const axis = keyCode === UP || keyCode === DOWN ? 'y' : 'x';
 		const value = parseFloat( next[ axis ] ) + delta;
 
-		next[ axis ] = roundClamp( value, 0, 1, step );
+		next[ axis ] = value;
```

Comment:
This is a runtime change in the context of this commit but the next commit restores the clamping. The rounding isn't needed here because the increments are fixed and applied to values that are already rounded.